cmd/config.go

```diff
@@ -47,12 +48,15 @@ var configurationSetCmd = &cobra.Command{
 	Long: `Set an individual value in the Odo configuration file
 Available Parameters:
 UpdateNotification - Controls if an update notification is shown or not (true or false)
-NamePrefix - Default prefix is the current directory name. Use this value to set a default name prefix.`,
+NamePrefix - Default prefix is the current directory name. Use this value to set a default name prefix.
+Timeout - Timeout (in seconds) for openshift server connection check`,
 	Example: `
    # Set a configuration value
    odo utils config set UpdateNotification false
    odo utils config set NamePrefix ""
    odo utils config set NamePrefix "app"
+   odo utils config set timeout 20
+   odo utils config set timeout 0
```

Comment:
same as this line, the line above is a good example, no need to show another example with just a different value